lib/ansible/module_utils/urls.py

```diff
@@ -590,6 +592,8 @@ def get_ca_certs(self):
         paths_checked.append('/etc/ansible')
 
         tmp_fd, tmp_path = tempfile.mkstemp()
+        to_add_fd, to_add_path = tempfile.mkstemp()
```

Comment:
Yes. The reason for that is that the tmp_ca_cert_path is used in more places, and is expected to contain everything for further underlying libraries. This way, I can limit the specific list that we add to OpenSSL to non-previously-loaded without limiting the others.